packages/file/lib/src/backends/memory/memory_directory.dart

```diff
@@ -47,6 +48,7 @@ class MemoryDirectory extends MemoryFileSystemEntity
 
   @override
   void createSync({bool recursive = false}) {
+    fileSystem.opHandle(path, FileSystemOp.create);
```

Comment:
Do we not have any tests where `createTempSync` fails?

Reply:
I only took a cursory glance, but didn't see any. I could update this PR to include create for that operation ez pz?

Reply:
Oh, the memory file system doesn't create the directory when you call createTemp - I'll fix that too.